src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -1641,12 +1652,15 @@ public async Task DataSetFileMetaCorrectlyReturned_Success()
                 dataSetFileMetaViewModel.GeographicLevels
                     .AssertDeepEqualTo(originalMeta!.GeographicLevels);
 
-                dataSetFileMetaViewModel.TimePeriod.AssertDeepEqualTo(
-                    new DataSetFileTimePeriodViewModel
+                dataSetFileMetaViewModel.TimePeriodRange.AssertDeepEqualTo(
```

Comment:
Don't need `AssertDeepEqualTo` as we're using records, can use standard `Assert.Equal`